src/bun.js/test/expect.zig

```diff
@@ -2649,6 +2649,57 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toEqualIgnoringWhitespace(this: *Expect, globalThis: *JSGlobalObject, callFrame: *CallFrame) callconv(.C) JSValue {
+        defer this.postMatch(globalThis);
+
+        const thisValue = callFrame.this();
+        const _arguments = callFrame.arguments(1);
+        const arguments: []const JSValue = _arguments.ptr[0.._arguments.len];
+
+        if (arguments.len < 1) {
+            globalThis.throwInvalidArguments("toEqualIgnoringWhitespace() requires 1 argument", .{});
+            return .zero;
+        }
+
+        active_test_expectation_counter.actual += 1;
+
+        const expected = arguments[0];
+        const value: JSValue = this.getValue(globalThis, thisValue, "toEqualIgnoringWhitespace", "<green>expected<r>") orelse return .zero;
+
+        const not = this.flags.not;
+        var pass = value.isString() and expected.isString();
+
+        if (pass) {
+            var expectedStr = expected.toString(globalThis).toSlice(globalThis, default_allocator).slice();
+            var valueStr = value.toString(globalThis).toSlice(globalThis, default_allocator).slice();
+
+            // Remove all whitespace from both strings
+            expectedStr = strings.removeWhitespace(default_allocator, expectedStr);
```

Comment:
Could you change this to compare the original strings, skipping all whitespace characters in the comparison? This way we don't need to allocate more memory